### PR TITLE
DOC-1546: Move the paste plugin details to core and removed options that no longer exist

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: DOC-1546

Description of Changes:
* Added the new `Copy & paste options` page to the `Content` section of the docs
* Moved the relevant paste plugin options to the new page
* Moved the relevant paste commands/menu items/toolbar buttons to the relevant core pages
* Removed `paste` from all the demos or code snippets
* Deleted the old pages for the options that were deleted
* Updated the options/commands that were kept to include changes made in 6.0

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -289,7 +290,6 @@
 *** PowerPaste
 **** xref:introduction-to-powerpaste.adoc[Introduction]
 **** xref:powerpaste-options.adoc[Options]
-**** xref:powerpaste-toolbars-menus.adoc[Toolbar buttons and menu items]
```
Comment (metricjs) [inline modules/ROOT/nav.adoc:292]: Just checking this was intentional?

Comment (lnewson) [reply modules/ROOT/nav.adoc:292]: Yes, all the UI components are in core now.

File: modules/ROOT/pages/available-menu-items.adoc [hunk 2]
```diff
@@ -155,26 +155,12 @@ include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
...
-:altplugincode: nil
-include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
-
 :plugincategory: premium
 :pluginname: Permanent Pen
 :plugincode: permanentpen
 :altplugincode: nil
 include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 
-:plugincategory: premium
```
Comment (metricjs) [inline modules/ROOT/pages/available-menu-items.adoc:170]: Checking this too?

Comment (lnewson) [reply modules/ROOT/pages/available-menu-items.adoc:170]: See the previous comment... the UI components are no longer in PowerPaste.

**Review by EkimChau — APPROVED** [review]

**Review by thedrjim — APPROVED** [review]

File: modules/ROOT/pages/copy-and-paste.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
+= Copy & paste options
+
+:title_nav: Copy and paste options
+:description: TinyMCE copy and paste
+
```
Comment (metricjs) [inline modules/ROOT/pages/copy-and-paste.adoc:5]: TinyMCE ships with copy and paste functionality that helps clean up content by default. In addition, the TinyMCE PowerPaste plugin adds extra functionality and control. 

^ some kind of intro like that? I can't think of the right words

File: modules/ROOT/pages/copy-and-paste.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+== {productname} PowerPaste plugin
+
+The {productname} PowerPaste plugin is a premium feature that provides improved paste support, including support for Microsoft Word, Microsoft Excel and Google Docs content. It also includes the option to clean or merge the styles when pasting content into the editor.
+
+For information, see the xref:introduction-to-powerpaste.adoc[PowerPaste plugin].
+
+== Controlling how content is pasted
+
+These options affect the way HTML content is processed when pasted in to the editor.
```
Comment (metricjs) [inline modules/ROOT/pages/copy-and-paste.adoc:14]: Are these a mix of core paste and powerpaste settings? Can't remember

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:14]: Some are, some aren't... I don't think we should mention anything here though as we have PowerPaste specific details in the PowerPaste plugin section of the docs.

File: modules/ROOT/pages/copy-and-paste.adoc (hunk 3 above)
Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: We shouldn't do that, as no other page here does that. If we must have it then it needs to be done on other pages and that's definitely out of scope.

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: I'm happy to rework the PowerPaste section below if needed, but I definitely don't want an intro section as that would make this page very inconsitent.

Comment (metricjs) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: Hm, fair. I couldn't think of another way to make it clearer that there was powerpaste and core paste, using the current structure is all. Oh. Unless the PowerPaste section begins with "TinyMCE has paste functionality built in, but the PowerPaste plugin..." kind of thing? I would like the following headings to be followed by something that clarifies what belongs to powerpaste and what's paste too, but words really were failing me

File: modules/ROOT/pages/copy-and-paste.adoc (hunk 4 above)
Comment (metricjs) [reply modules/ROOT/pages/copy-and-paste.adoc:14]: None of these are PowerPaste specific at least, right? I guess that's more what I was thinking - clarifying that these settings are part of TinyMCE core and that you don't need PowerPaste for them

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:14]: Yeah, none are PowerPaste specific.

File: modules/ROOT/pages/copy-and-paste.adoc (hunk 3 above)
Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: Like I said, if we're doing that then we need to do it elsewhere as well. I'm more than happy to change the PowerPaste section as much as needed and starting with `TinyMCE has paste functionality built in, but the PowerPaste plugin...` sounds like the a good idea there.

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: Another issue I realised with including that type of message below, is some options do also apply to PowerPaste (e.g. `paste_as_text`, `paste_tab_spaces`, `paste_preprocess` and `paste_postprocess`). As such, including that would likely only lead to confusion.

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: I've reword the PowerPaste spiel at the top to start with something like the above for now.

Comment (lnewson) [reply modules/ROOT/pages/copy-and-paste.adoc:5]: Added the admonition to try and clarify this as discussed on slack.

**Review by metricjs — APPROVED** [review]

**Pull request merged**